espresso-server/app/proguard-rules.pro

```diff
@@ -2,7 +2,7 @@
 # By default, the flags in this file are appended to flags specified
 # in ~/Library/Android/sdk/tools/proguard/proguard-android.txt
 # You can edit the include path and order by changing the proguardFiles
-# directive in build.gradle.
+# directive in build.gradle.kts.
```

Comment:
extra dot at he end

Reply:
I know, I just kept the dot from before because it's part of the sentence. Should I really remove it?

Reply:
up to you